### PR TITLE
Allow testing on still supported versions and completely disable it on older versions

### PR DESCRIPTION
**Pull request by pzhlkj6612**
Checklist:

- [x] Allow testing on still supported versions (5.12+).
- [x] Warn users that the tests are disable.
- [x] Completely disable tests if `CONFIG` contains `WITHOUT_TESTS` .

### CODE REVIEW

File: CQtDeployer.pro [hunk 1]
```diff
@@ -8,8 +8,8 @@
 TEMPLATE = subdirs
 CONFIG += ordered
 
-lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 14) {
-    message(Tests is disabled!)
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 12) {
+    warning("Tests are only enabled on Qt 5.12.0 or later version. You are using $$[QT_VERSION].")
     DEFINES += WITHOUT_TESTS
 }
```
Comment (EndrII) [inline CQtDeployer.pro:14]: please add the  
```
include($$PWD/test.pri)
```
in to contains block WITHOUT_TESTS
```
!contains(DEFINES, WITHOUT_TESTS) {
```

**Review by EndrII — CHANGES_REQUESTED** [review]

**Review by EndrII — APPROVED** [review]
Okay i think it is ready to merging

File: CQtDeployer.pro (hunk 1 above)
Comment (pzhlkj6612) [reply CQtDeployer.pro:14]: Well, I did this in another way. Please see https://github.com/QuasarApp/CQtDeployer/compare/cae9981e5bf62ccd13c1110ff3ddfc5a614448cf...ff9690b1155d40355a3adbe433cc9cf080298e5b .

**Comment by EndrII** [comment]
@pzhlkj6612 This branch is ready to merging ?

**Pull request merged**